apis/bases/network.openstack.org_netconfigs.yaml

```diff
@@ -55,6 +55,10 @@ spec:
                         ...
                       pattern: ^[a-zA-Z0-9][a-zA-Z0-9\-_]*[a-zA-Z0-9]$
                       type: string
+                    serviceNetwork:
+                      description: Service network mapping
+                      pattern: ^[a-zA-Z0-9][a-zA-Z0-9\-_]*[a-zA-Z0-9]$
```

Comment:
should the servicenetwork forced to be all lower case, and document the internal defaults (ctlplane, internalapi, external, storage, storagemgmt and tenant)?

Reply:
Network and hence SeviceNetwork are case insensitive as before. Ctlplane and ctlplane mean the same. It's handled in code.

Reply:
What benefits do you think we would get with that? We already allow case insensitivity of `Network` after https://github.com/openstack-k8s-operators/infra-operator/commit/0cab900cc4025768b5b48fda396efb1ee5f92fd1 and never made it case-sensitive. We can allow them to be used as ServiceNetwork without case conversion in defaulting. Also, we  use mixed case in many places like https://github.com/openstack-k8s-operators/infra-operator/blob/main/config/samples/network_v1beta1_ipset.yaml#L7

Reply:
I was just wondering if it makes sense to force the user to use lowercase for the service network, otherwise the user might wonder why different service network name would be used when have to debug. right now you'd have to know that it will be made lowercase by the code. that was just what I was thinking.

Reply:
ah wait, the servicenetwork in the reservation will be automatic lower case. I think we are good. sorry